scipy/stats/_continuous_distns.py

```diff
@@ -4662,12 +4662,13 @@ class t_gen(rv_continuous):
 
     .. math::
 
-        f(x, df) = \frac{\gamma((df+1)/2)}
-                        {\sqrt{\pi*df} \gamma(df/2) (1+x^2/df)^{(df+1)/2}}
+        f(x; \text{df}) = \frac{\Gamma((\text{df}+1)/2)}
+                        {\sqrt{\pi*\text{df}} \Gamma(\text{df}/2)}
+                    (1+x^2/\text{df})^{-(\text{df}+1)/2}
```

Comment:
Possibly rename the parameter to \nu in the formula and mention in the text that it corresponds to `df` in code?